rfc/0014-attribute-based-valuetypes/README.md

```diff
@@ -42,47 +37,61 @@ valuetype CorrelationCoefficient oftype decimal {
 }
 ```
 
-This syntax is smart in that you don't have to list and name an attribute but rather rely on an implicit 'value' attribute. Still, I propose to make that attribute explicit. New syntax would be:
+This syntax is smart in that you don't have to list and name an attribute but rather rely on an implicit 'value' attribute.
+
+The current syntax, in which an underlying value type is referenced through 'oftype', it does not use instantiation/inheritance but rather composition.
+The purpose of using composition is that it is
+
+1. more similar to traditional approaches and
```

Comment:
What is a traditional approach? Since you mentioned inheritance before, does it correlate to that?

Reply:
I don't know, I just took it from a previous version :)

Reply:
Reformulated it a little bit